modules/auth/webauthn/webauthn.go

```diff
@@ -0,0 +1,21 @@
+package webauthn
+
+import (
+	"github.com/duo-labs/webauthn/webauthn"
+
+	"code.gitea.io/gitea/modules/setting"
+)
+
+//WebAuthn represents the global WebAuthn instance
+var WebAuthn *webauthn.WebAuthn
+
+//Init initializes the WebAuthn instance from the config.
+func Init() {
```

Comment:
I'm not 100% happy with this. I would prefer to use webauthn.New() in order to validate the settings, but this routers.GlobalInit() seems to be the best place to put this and there is no error handling there. Open to suggestions.